DuckDuckGo/AuthConfirmationPromptView.swift

```diff
@@ -40,7 +40,7 @@ struct AuthConfirmationPromptView: View {
                 Button {
                     viewModel.cancelButtonPressed()
                 } label: {
-                    Text(UserText.autofillCancel)
+                    Text(UserText.actionCancel)
```

Comment:
It turns out `UserText.autofillCancel` in not actually translated, so updated to use `UserText.actionCancel ` which is correctly translated